target/linux/ath79/image/generic.mk

```diff
@@ -488,6 +509,21 @@ define Device/avm_fritzdvbc
 endef
 TARGET_DEVICES += avm_fritzdvbc
 
+define Device/avm_fritzx490-wasp
+  SOC := qca9558
+  DEVICE_VENDOR := AVM
+  DEVICE_MODEL := FRITZ!Box 3490/5490/7490 WASP (Wireless Assist)
+  KERNEL := kernel-bin | append-dtb | relocate-kernel | lzma-no-dict | \
+      avm-wasp-loader | pad-to 4096 | pad-extra 208 | avm-wasp-checksum x490
+  LOADER_TYPE := bin
+  KERNEL_INITRAMFS := $$(KERNEL)
+  DEVICE_PACKAGES := kmod-ath10k-ct ath10k-firmware-qca988x-ct-full-htt \
+	wpad -wpad-basic-wolfssl -uboot-envtools -swconfig -ppp \
```

Comment:
Why do you use `wpad` instead of the default `wpad-basic-wolfssl` ? 
Why do you use `ath10k-firmware-qca988x-ct-full-htt` instead of ath10k-firmware-qca988x-ct`?